package.json

```diff
@@ -42,6 +42,7 @@
     "needle": "^2.3.3",
     "pdfreader": "^1.0.6",
     "puppeteer": "^2.1.1",
+    "selenium-webdriver": "^4.0.0-alpha.7",
```

Comment:
I've had flakiness with selenium at the best of times. Do we as a project
a) want selenium when we already have puppeteer? and
b) want an alpha of selenium which is likely even more flaky than usual?

Reply:
Yes, highly recommend we use Puppeteer -- we can't take on another large dep as we move to cloud functions.

Reply:
It works, that is it gets the right data, but it runs very slow when I set the browser to headless.  I'll look into puppeteer and see what I can come up with.

thanks for the input.